diff --git a/es/TabNavList/TabNode.js b/es/TabNavList/TabNode.js
--- a/es/TabNavList/TabNode.js
+++ b/es/TabNavList/TabNode.js
@@ -90,7 +90,7 @@ const TabNode = /*#__PURE__*/React.forwardRef((props, ref) => {
       overflow: 'hidden',
       opacity: 0
     }
-  }, `Tab ${currentPosition} of ${tabCount}`), icon && /*#__PURE__*/React.createElement("span", {
+  }, `${currentPosition} / ${tabCount}`), icon && /*#__PURE__*/React.createElement("span", {
     className: `${tabPrefix}-icon`
   }, icon), label && labelNode), removable && /*#__PURE__*/React.createElement("button", {
     type: "button",
diff --git a/lib/TabNavList/TabNode.js b/lib/TabNavList/TabNode.js
--- a/lib/TabNavList/TabNode.js
+++ b/lib/TabNavList/TabNode.js
@@ -98,7 +98,7 @@ const TabNode = /*#__PURE__*/React.forwardRef((props, ref) => {
       overflow: 'hidden',
       opacity: 0
     }
-  }, `Tab ${currentPosition} of ${tabCount}`), icon && /*#__PURE__*/React.createElement("span", {
+  }, `${currentPosition} / ${tabCount}`), icon && /*#__PURE__*/React.createElement("span", {
     className: `${tabPrefix}-icon`
   }, icon), label && labelNode), removable && /*#__PURE__*/React.createElement("button", {
     type: "button",
